project/src/components/Form.js

```diff
@@ -0,0 +1,251 @@
+import React, {Component} from 'react'
+import { connect } from 'react-redux'
+import { addValue } from '../actions/index'
+import {withRouter} from 'react-router-dom'
+import $ from 'jquery'
+
+import '../../node_modules/bootstrap/dist/css/bootstrap.min.css';
+import '../css/style.css';
+const Validation = require ('../components/validationComponent.js');
+const {addUserUrl} = require('../constants');
+
+const myRouterComponent = withRouter (class Form extends Component {
+
+    render(){
```

Comment:
In render function, you have only to return the result. Probably, some additional logic connected to rendering.
Do not store in render function your business logic - handlers, additional checks and so on. You can create class methods instead.